Draft.md

```diff
@@ -156,235 +156,171 @@ Eine Personengruppe, die besonders oft zum passiven Spielball von Forschungssoft
 
 ## Mögliche Vorgehensweise zur Rezension archäologischer Forschungssoftware
 
-Nach unserem Verständnis sollte eine Rezension archäologischer Software, so wie es eine herkömmliche Besprechung einer wissenschaftlichen Publikation leistet, die Software zunächst vorstellen, dabei zusammengefasst Eckdaten vorlegen und den Kontext erläutern. Im Anschluss sollte eine kritische Beurteilung erfolgen.
+Nach unserem Verständnis sollte eine Rezension archäologischer Software, so wie es eine herkömmliche Besprechung einer wissenschaftlichen Publikation leistet, die Software zunächst vorstellen, dabei zusammengefasst Eckdaten vorlegen und den dann den ausführlicheren Kontext erläutern. Schließlich sollte eine kritische Beurteilung erfolgen.
 
-Umgesetzt auf Software gehören zu den Eckdaten Angaben zur Version und den Release-Daten, zu den Entwicklern, der Lizenz und Weiteres. Als Kontext verstehen wir eine erste Einordnung in das archäologische Forschungsfeld und Angaben zum möglichen Zusammenhang mit Forschungsprojekten, Arbeitsgruppen oder Institutionen. Für die kritische Beurteilung aus unterschiedlicher Perspektive haben wir unten einen Katalog an Fragen zusammengestellt. Zuletzt sollten die Ergebnisse der Betrachtung in einer Stellungnahme zusammengefasst werden, die die Software hinsichtlich ihrer Nützlichkeit, ihrer Bedienbarkeit, ihrer handwerklichen Qualität und ihrer Position in Relation zu den Idealen guter Forschungssoftware (z.B. FAIR und CARE), beurteilt.
+Umgesetzt auf Software gehören zu den Eckdaten Angaben zur Version, zu den Entwicklern, der Lizenz und Weiteres. Wir schlagen vor, die Rezension mit einer tabellierten Übersicht wie der folgenden einzuleiten:
 
```

Comment:
Das geht mir zu weit, und zwar sowohl:
- Tabelle
- Position der Tabelle am Beginn
- Das, was hier als Eckdaten gezählt wird.

Reply:
Nun - das war mal ein erster Entwurf. Aber: 
- Wir hatten doch in der Gruppe besprochen, dass wir eine solche Tabelle (!) möchten? Ich mache das ja nicht zum Spaß :smile: 
- Wenn wir wirklich vorschlagen möchten, dass eine solche Tabelle  am Beginn jeder Softwarerezension stehen soll, dann muss sie doch promininent platziert werden?
- Welche Eckdaten würdest du denn konkret weglassen? Wie wäre es mit folgender Minimalfassung:
  - Name + Version
  - Plattform
  - URL

Finde ich weniger nützlich, aber ich habe keine starke Meinung.

Reply:
Pardon, ich habe immer im Kopf, dass die Tabelle ans Ende kommt und wir es auch als eine Art Zusammenfassung verstehen. Aber eigentlich auch egal: Für mich sollten unsere Impulse klar von Redaktionsrichtlinien getrennt werden. Daher sollten wir vielleicht adressieren, dass eine solche Tabelle sehr sinnvoll wäre, um ..... Wo sie steht, sollten wir dann vielleicht ganz bewusst als Entscheidung der Redaktion darstellen. 

```suggestion
Wir schlagen vor den Lesern diese Eckdaten in einer tabellarischen Übersicht zur Verfügung zu stellen. Dies ermöglicht z. B. eine schnelle Übersicht, darüber ob die behandelte Software mit den eigenen technischen Systemen und zusammenarbeitet.  
```